util/src/dbutil.cpp

```diff
@@ -224,5 +281,32 @@ std::shared_ptr<odb::database> createDatabase(const std::string& connStr_)
   return db;
 }
 
+void createTables(
+  std::shared_ptr<odb::database> db_,
+  const std::string& sqlDir_)
+{
+  runSqlFiles(db_, sqlDir_,
+    [](const std::string& s){
+      return removeByRegex(removeByRegex(removeByRegex(s,
+        "CREATE UNIQUE INDEX", ";"),
+        "CREATE INDEX", ";"),
+        "ALTER TABLE", ";");
+    },
+    "Creating tables from file ");
```

Comment:
Write the space character at the end of message in the `runSqlFiles()` function:
```cpp
LOG(info) << logMessage_ << " " << fileName;
```